dev-support/Jenkinsfile_GitHub

```diff
@@ -102,11 +104,11 @@ pipeline {
                             checkout scm
                         }
                         dir("${YETUSDIR}") {
-                            checkout([
-                              $class           : 'GitSCM',
-                              branches         : [[name: "${YETUS_VERSION}"]],
```

Comment:
IIRC, this previous behavior was so that we could test with upstream yetus feature branch if need be. Maybe unrealistic...

Reply:
We have a special option for nightly job, for using the newest yetus code.

I think later we could introduce the same logic for pre commit.